routes/auth/guest/getGuests.ts

```diff
@@ -1,28 +1,31 @@
-import { db, event } from '#src/utils'
+import { db, event } from "#src/utils";
 
 export const getGuests = async () => {
-    const limit = event.body?.limit ?? 10;
-    const offset = event.body?.offset ?? 0;
-    let first_name = event.body?.first_name && `%${event.body.first_name}%`;
-    let last_name = event.body?.last_name && `%${event.body.last_name}%`;
-    let dob = event.body?.dob && `%${event.body.dob}%`;
-    let guest_id = event.body?.guest_id;
-    const query = event.body?.query && `%${event.body.query}%`;
+  const limit = event.body?.limit ?? 10;
+  const offset = event.body?.offset ?? 0;
+  let first_name = event.body?.first_name && `%${event.body.first_name}%`;
+  let last_name = event.body?.last_name && `%${event.body.last_name}%`;
+  let dob = event.body?.dob && `%${event.body.dob}%`;
+  let guest_id = event.body?.guest_id;
+  const query = event.body?.query && `%${event.body.query}%`;
+  const sort = event.body?.sort === "desc" ? "DESC" : "ASC";
```

Comment:
Not sure of the ramifications of using event.body vs event.queryStringParamaters. Maybe they're interchangeable, but I'd hope that using body doesn't affect the ability to encode the sort and offset in the url?